ocaml/xapi/helpers.ml

```diff
@@ -1082,6 +1082,10 @@ let assert_using_vswitch ~__context =
   in
   if not using_vswitch then raise Api_errors.(Server_error (openvswitch_not_active, []))
 
+let assert_pvs_servers_available ~__context ~pvs_site =
+  let pvs_servers = Db.PVS_site.get_servers ~__context ~self:pvs_site in
```

Comment:
Yes, I considered that but decided to follow the same style as the 2 functions above this one.